docker/airflow/1.10.3/onbuild/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM astronomerinc/ap-airflow:0.9.0-alpha.2-1.10.2
+FROM astronomerinc/ap-airflow:0.9.0-alpha.1-1.10.3
```

Comment:
@andriisoldatenko you can revert this change. We will automatically update this tag as part of the release process (https://github.com/astronomer/astronomer/blob/master/Makefile#L144-L148).